core/src/test/scala/com/rackspace/com/papi/components/checker/wadl/WADLCheckerSpec.scala

```diff
@@ -6996,7 +7195,7 @@ class WADLCheckerSpec extends BaseCheckerSpec with LogAssertions {
              XML.loadFile("src/test/resources/xsd/test-urlxsd.xsd"))
     When("the wadl is translated")
     val checker = builder.build (inWADL, TestConfig(false, false, true, true, true, 1,
-                                                    true, true, true, "XalanC",
+                                                    true, false, true, "XalanC",
```

Comment:
...because that's not the purpose of the test, it didn't make difference before to set XSD Grammar Transform to true, but now it does.